docs/content/docs/contributor-docs/release.md

```diff
@@ -103,11 +108,12 @@ referring to the release you've just built artifacts for.
 4) Commit your code and create a pull request:
 ```bash
 git commit -a -m "release artifacts for release $RELEASE_VERSION"
+git push origin HEAD
 ```
 
-5) Get your pull request reviewed and merged.
+5) Get your pull request reviewed and merged. After merge, tag is automatically applied and pushed **unless v0.0.1 release**
```

Comment:
Please remove the exception for v0.0.1 release in this doc.
I have pushed the bug fix for that.
https://github.com/aws-controllers-k8s/test-infra/pull/159#issuecomment-976089983